test/SILGen/objc_bridging_any.swift

```diff
@@ -1,763 +0,0 @@
-// REQUIRES: plus_one_runtime
```

Comment:
Did you mean to completely delete this test?

Reply:
I kept it in the master and 4.1 branches, but it has branch conflicts that needed to be resolved in yet another way in 4.2. Since 4.2 is rebranching soon and is not that different from master it didn't seem worth the time to deal with the conflict here.